### DISCUSSION TITLE
How to make 'Focus Overlapped Terminal' available

### DISCUSSION THREAD

**Discussion by astr0gator** [mikekazakov/nimble-commander#442] [Q&A]
![image](https://github.com/user-attachments/assets/f186c193-fb14-4274-9b7e-39375115362d)

**Comment by mikekazakov** [comment]
This hotkey becomes enabled when the overlapped terminal is visible.
Use the "Panel Position > Move Up" hotkey (Ctrl+Alt+Up) to move the panels up  and reveal the overlapped terminal, e.g. like this:
<img width="752" alt="Screenshot 2024-11-09 at 11 57 09" src="https://github.com/user-attachments/assets/b1c0fa1d-73f1-4b4e-acb7-6ee12951d73a">

**Comment by astr0gator** [comment]
Thank you, that was my guess too. However, my terminal (launched with 'Show terminal') is taking the whole window and resizing just resizes the terminal window (NC is panels are not available)

UPD: Version 1.6.1 (4088), standalone

**Reply by mikekazakov** [reply]
There are two different terminal modes available in NC at the same time - the one that takes the whole window (shown by alt+cmd+o) and the overlapped one that allows file panels to be shown on top of it (shown by e.g. ctrl+alt+o or ctrl+alt+up).

**Comment by astr0gator** [comment]
Thank you, very helpful
Without magic secret combo of ctrl+alt+o (covers the whole screen!) AND ctrl+alt+up (uncover terminal line by line) I would't find it.
Help shows only one option of terminal. 

<img width="617" alt="image" src="https://github.com/user-attachments/assets/db5c88e7-f071-4c47-ae0c-bd6785d709d1">

**Reply by mikekazakov** [reply]
This hotkey is available in the menu under `View > Panel Position > Hide Panels`.

There's no mechanism to import settings from other software into NC.

Though this screenshot looks very strange - which theme is that?
Which modifications were done to it?

**Comment by astr0gator** [comment]
Ok, now I understand the logic of terminal/panes interaction. Thanks

Settings: It uses my default iterm2 settings, works great. 
Artefacts: they were produced by changing the font size for terminal using 'standard' font size, Menlo works great though.

Everything solved. Thank you once again, the best app on my mac 🙂
